eva/third_party/huggingface/model.py

```diff
@@ -48,3 +49,30 @@ def input_formatter(self, inputs: Any):
         frames = np.vstack(frames_list)
         images = [Image.fromarray(row) for row in frames]
         return images
+
+
+class AudioHFModel(AbstractHFUdf):
+    """
+    Base Model for all HF Models that take in audio as input
+    """
+
+    def input_formatter(self, inputs: Any):
+        audio = np.empty((0, 0))
+        files = set(inputs.iloc[:, 0].tolist())
+        for file in files:
+            reader = decord.AudioReader(file, mono=True, sample_rate=16000)
+            audio = np.append(audio, reader[0:].asnumpy()[0])
+        return audio
```

Comment:
This is necessary because most models are trained with a 16000 sampling rate. In the future we could maintain a map of model to sampling rate